python/paddle/v2/fluid/nets.py

```diff
@@ -38,8 +41,10 @@ def img_conv_group(input,
                    param_attr=None,
                    conv_with_batchnorm=False,
                    conv_batchnorm_drop_rate=None,
+                   conv_use_cudnn=True,
```

Comment:
`conv_use_cudnn`, `pool_use_cudnn` in argument is not flexible enough.
I think we can expose the 
`HasCUDNN` DSO function,  HasCUDNN will determine use cudnn or not according to the machine environment.